signature/provider.go

```diff
@@ -0,0 +1,176 @@
+package signature
+
+import (
+	"context"
+	"crypto"
+	"crypto/x509"
+	"fmt"
+
+	"github.com/notaryproject/notation-core-go/signature"
+	"github.com/notaryproject/notation-go/plugin"
+)
+
+// builtInPluginMetaData is the builtin metadata used by builtinProvider.
```

Comment:
nit: I feel `is the metadata used by builtinProvider.` is enough.

Reply:
Updated